server/command.go

```diff
@@ -142,21 +150,29 @@ func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User, topi
 		}
 		return authErr.Message, authErr.Err
 	}
-
-	client, _, err := p.getActiveClient(user)
-	if err != nil {
-		p.API.LogWarn("Error creating the client", "err", err)
-		return "Error creating the client.", nil
+	var meetingID int = -1
+	var createMeetingErr error = nil
+
+	if userPMISettingPref, getUserPMISettingErr := p.getPMISettingData(user.Id); getUserPMISettingErr == nil {
+		switch userPMISettingPref {
+		case zoomPMISettingValueAsk:
+			p.askUserPMIMeeting(user.Id, args.ChannelId)
+		case "", trueString:
```

Comment:
please explain what is this we are checking userPMISettingPref to empty or trueString

Reply:
Let say till now user have not set the preference whether he will create a meeting with PMI or not. Then in that case we are starting the meeting with PMI